### DISCUSSION TITLE
Is there a way to revert to older notebooks?

### DISCUSSION THREAD

**Discussion by THX1139b** [TheLastBen/fast-stable-diffusion#1594] [Q&A] [answered]
I have made around 50 models now for SD1.5 in total but lately I am no longer successful. I think the last really good .ckpt I created with Dreambooth was in the beginning of January using 18 old photos. Since then everything has gotten worse. Now my finished characters either look a bit zombiefied or they simply don't have a great likeness. I think training is a lot faster but my results are really bad compared to before. It seems I have tried just about everything in settings.

Today as a test I have tried using an old set of 40 training photos (from last year when people recommended using a lot of images), -a set that gave great results before but now the results are bad. I tried redusing the number of images first to 20 and then to 10. I have also tried with and without captions and I have tried other settings than the default ones but it seems hopeless. Nothing gets me results like before the latest changes.

**Comment by TheLastBen** [comment]
set the unet learning rate to 2e-6, that's the only thing that changed

**Reply by THX1139b** [reply]
OK, thanks. I have tried that but it doesn't work for me anymore. Oh well.

**Comment by Mossbraker** [comment, marked as answer]
You should be able to go to File > Revision History, or something

**Reply by THX1139b** [reply]
I didn't know about that feature, thanks!
